conda/recipes/numbast/meta.yaml

```diff
@@ -20,32 +20,25 @@ build:
   script:
     - {{ PYTHON }} -m pip install . -vv
   ignore_run_exports_from:
-    - {{ compiler('cuda') }}
     - libnvjitlink-dev
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}
     - cmake >=3.28
-    - pybind11
-    - cffi
-    - clangdev >=18
     - ninja
     - ast_canopy
```

Comment:
`ast_canopy` should be in `host` (if it provides headers/libraries needed to compile numbast) or `run` (if only needed at runtime).

Reply:
`ast_canopy` is only needed at runtime.